Angular InstantSearch/conditional-request/karma.conf.js

```diff
@@ -9,23 +9,36 @@ module.exports = function (config) {
       require('karma-jasmine'),
       require('karma-chrome-launcher'),
       require('karma-jasmine-html-reporter'),
-      require('karma-coverage-istanbul-reporter'),
+      require('karma-coverage'),
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client: {
+      jasmine: {
+        // you can add configuration options for Jasmine here
+        // the possible options are listed at https://jasmine.github.io/api/edge/Configuration.html
+        // for example, you can disable the random execution with `random: false`
+        // or set a specific seed with `seed: 4321`
+      },
```

Comment:
Should we keep this?

Reply:
I'd say yes, if we don't, it's reappear next time we update, and I prefer not having to deal with too many diff.
It's part of the Angular CLI template.